libraries/AP_HAL_ChibiOS/hwdef/SkySakuraH743/defaults.parm

```diff
@@ -0,0 +1,23 @@
+# SERIAL1 is MAVLink2 by default
```

Comment:
delete all items.....they are in hwdef to not consume flash....only these are needed:
SERVO13_FUNCTION 120
NTF_LED_TYPES 257